kubernetes/provider.go

```diff
@@ -181,6 +208,18 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 		cfg.BearerToken = v.(string)
 	}
 
+	if v, ok := d.GetOk("exec"); ok {
+		exec := &clientcmdapi.ExecConfig{}
+		spec := v.(*schema.Set).List()[0].(map[string]interface{})
```

Comment:
Also, would you mind breaking down this line into multiple steps and check for errors and nils on the type assertions. This looks like it has the potential to panic.

Reply:
In theory, it should be pass by the schema checker, but I've added an extra check to avoid the panic